### PR TITLE
WIP-RIL-377 Fix for buildkite/puppeteer:8.0.0 docker image incompatible with Node 18

### PR DESCRIPTION
**Pull request by SavantDhindsaHO**
## What?
Update project to Node version 18. see jira ticket #RIL-376.
https://collaboration.homeoffice.gov.uk/jira/browse/RIL-376
## Why?
Part of Epic to upgrade all HOF services to Node v18. see jira ticket #HOFF-312 and dependedncy for RIL-336
https://collaboration.homeoffice.gov.uk/jira/browse/HOFF-312
https://collaboration.homeoffice.gov.uk/jira/browse/RIL-336
## How?
- Updated node engine inside package.json from 14.15.0 to 18.12.1
- Updated main docker image to node:lts-alpine
- Changed all references to node:14 in drone.yml to node:18
## Anything Else?

### CODE REVIEW

File: README.md
Comment (ChisomOguibe11) [inline README.md:8]: I think this was meant to say nvm use 18.12.1

File: README.md
Comment (ChisomOguibe11) [inline README.md:124]: This comment can be removed now.

File: test/_accessibility/accept/accessibility.spec.js
Comment (ChisomOguibe11) [inline test/_accessibility/accept/accessibility.spec.js:54]: Best to remove all console.logs before merging

File: test/_accessibility/accept/accessibility.spec.js
Comment (ChisomOguibe11) [inline test/_accessibility/accept/accessibility.spec.js:82]: Can be removed also

**Review by ChisomOguibe11 — CHANGES_REQUESTED** [review]
Looks good. Well done on getting the accessibility tests running! 🙌🏾

I'll approve once all the console.logs are removed

File: README.md
Comment (SavantDhindsaHO) [reply README.md:8]: yes, Thanks for noticing. I have updated it.

File: README.md
Comment (SavantDhindsaHO) [reply README.md:124]: Removed.

File: test/_accessibility/accept/accessibility.spec.js
Comment (SavantDhindsaHO) [reply test/_accessibility/accept/accessibility.spec.js:54]: I have added these console log to group pa11y result and page information together and it will help to debug if any pa11y result show any issue

**Review by ChisomOguibe11 — APPROVED** [review]
lgtm

File: test/_accessibility/accept/accessibility.spec.js
Comment (SavantDhindsaHO) [reply test/_accessibility/accept/accessibility.spec.js:82]: need for debug

**Pull request merged**
